src/targets/betafpv_900.ini

```diff
@@ -0,0 +1,31 @@
+# ********************************
+# Transmitter targets
+# ********************************
+
+[env:BETAFPV_900_TX_via_UART]
+extends = env_common_esp32
```

Comment:
The `BETAFPV_2400` target seems to be different to this one? Is that intentional? I.e. the 2400 target has the line `board_build.f_cpu = 160000000L` but this one doesn't?